dietpi/dietpi-software

```diff
@@ -15140,6 +15168,13 @@ _EOF_
 			[[ -d '/var/lib/pijuice' ]] && G_EXEC rm -R /var/lib/pijuice
 
 		fi
+		
+		software_id=206 # Homer
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 ))
+		then	
+			Banner_Uninstalling
+			[[ -d '/var/www/homer' ]] && G_EXEC rm -R /var/www/homer
```

Comment:
good question. Usually, I would say yes because we don't keep any backup for one of our apps. Except Own-/Nextcloud.

Reply:
that'd be good yeah

Reply:
We also do not create backups in any other case, AFAIK. Ah, we can ask the user and print an info otherwise (also in case of non-interactive uninstall.

Reply:
let's keep it simple. If someone decide to uninstall, he needs to be clear everything will be purged.

Reply:
Too late 😄. It's fine, not too much code.

Reply:
ok let's do it that way.